.github/workflows/main-test.yml

```diff
@@ -62,7 +62,6 @@ jobs:
           - libraries/cli-graphics
           - libraries/logging
           - libraries/metric
-          - libraries/module
```

Comment:
I think we should not remove this from the workflow, nor should we replace the test scripts. Simply creating a test case would solve this problem too.

Reply:
In principle, I agree.

In #382 I replaced the manual specifications of which modules to use with an automatic mechanism, so this would partly solve this adding and removing of modules. However, this package already did not have any (real) test cases. I just disabled the module as otherwise the CI would fail.

Given I am planning to overhaul the module package as well, I don't think it makes sense to add test cases for the module package here, as it is unrelated to the changes in this PR. I also don't think adding a "fake" test case is a useful solution here.

Reply:
Alright, as long as they are added back in another PR.